src/cmd/manage_nsfs.js

```diff
@@ -882,41 +882,21 @@ function validate_no_extra_options(type, action, input_options) {
 }
 /**
  * validate_options_type_by_value check the type of the value that match what we expect.
- * in case that type of value we have is boolean (and the option type is not boolean) -
```

Comment:
do we have this check somewhere else?

Reply:
Not sure I understand the question...
The check for "type of value we have is boolean (and the option type is not boolean)" is part of the check of invalid type, I tried to separate this case to create a different error message (but it resulted in the issue I'm trying to fix now...), so I decided to drop it and send the same general error message in the details.